app/src/matrix_transform.c

```diff
@@ -11,17 +11,17 @@
 
 #ifdef ZMK_KEYMAP_TRANSFORM_NODE
 
-#define _TRANSFORM_ENTRY(i, _)                                                                     \
+#define INDEX_OFFSET 1
```

Comment:
This code is great, and clear *based on knowing the context of this PR*, but I do fear that next time this gets looked at in a few months it won't be as clear. I'm not a fan of "code comments for the sake of comments", but can you add a small bit of context/explanation above this define to clarify the offset is used so that only real values are non-zero, so the default 0 value is left as a "undefined transform mapping" to be detected.

Once that's in, this is good to go. Thanks!

Reply:
DONE, added comment.